src/MongoDB.Driver.Legacy/MongoServerSettings.cs

```diff
@@ -1191,19 +1191,17 @@ internal ClusterKey ToClusterKey()
             return new ClusterKey(
                 _allowInsecureTls,
                 _applicationName,
-                bypassQueryAnalysis: null, // not supported for legacy
                 _clusterConfigurator,
                 _compressors,
                 _connectionMode,
                 _connectionModeSwitch,
                 _connectTimeout,
                 _credentials.ToList(),
+                cryptClientSettings: null,
```

Comment:
let's also add this command as it was here before: `// not supported for legacy`